pages/common/xephyr.md

```diff
@@ -0,0 +1,11 @@
+# Xephyr
+
+> A nested X server that runs as an X application.
+
+- Create a black window with display ID ":2":
+
+`Xephyr -br -ac -noreset -screen 800x600 :2`
+
+- Start BSPWM on the new screen:
+
+`DISPLAY=:2 bspwm`
```

Comment:
`bspwm` is another windows manager and should have his own page.

Reply:
`DISPLAY=:2 bspwm` is an example of how to start an X application to the Xephyr server. 
Do you think it is unnecessary? Usually, you wouldn't use DISPLAY=:2 on startup. So neither BSPWM nor Xephyr would contain this example.

By the mentality, "When in doubt, keep new command-line users in mind. Err on the side of clarity rather than terseness. For example, commands that require sudo should include it directly in the examples", I thought there is a need to show people how to use that black screen that Xephyr gave.

Should I remove it?

Reply:
Hrm, this is a tricky one. Perahaps it would be best if we use a more generic example here. What about this?
```suggestion
`DISPLAY=:2 {{command_name}}`
```
The example description would also need updating.